apps/backend/src/project/project.service.ts

```diff
@@ -11,39 +12,39 @@ export class ProjectService {
     private readonly projectRepository: EntityRepository<ProjectEntity>
   ) {}
 
+  async create(projectDto: ProjectDto, owner: UserEntity, contributors: UserEntity[]): Promise<ProjectEntity> {
+    const project = new ProjectEntity(
+      projectDto.title,
+      new Date(),
+      projectDto.draft,
+      projectDto.datasets,
+      projectDto.description,
+      owner,
+      contributors,
+      projectDto.config,
+      projectDto.version
+    );
+    await this.projectRepository.persistAndFlush(project);
+    return project;
+  }
+
   async findAll(): Promise<ProjectDto[]> {
     return this.projectRepository.findAll();
   }
+
   async findOneById(id: number): Promise<ProjectDto> {
     return this.projectRepository.findOne({ id });
   }
 
-  async update(id: number, projectDto: ProjectDto): Promise<ProjectDto> {
+  async update(id: number, projectDto: ProjectDto): Promise<void> {
```

Comment:
An update function should most likely return the updated project instead of void.

Reply:
Is it an expected behavior ?
`Mikroorm` does not sends back the object after an update. This would require another call to the database. Although it is quite simple to do, I am a bit reluctant : the database appears slower and slower in my test files.
Of course, it it is an expected behavior, I'll do it. What do you think @lutangar  ?

Reply:
> Mikroorm does not sends back the object after an update. This would require another call to the database.

Yes, maybe this is not the right place to do this. Maybe we will only return the updated entity in a response of an update API call but maybe not here.

Reply:
@racine-p-a about this :
https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#useful-post-responses